app/serializers/item_serializer.rb

```diff
@@ -1,3 +1,9 @@
 class ItemSerializer < ActiveModel::Serializer
+  include ModelHelpers
+
   attributes :id, :name, :description, :unit_price, :merchant_id, :created_at, :updated_at
+
+  def unit_price
```

Comment:
i think you could almost define your penny_to_dollar method here. what's the purpose of this method? who needs access to it?